.github/workflows/test.yml

```diff
@@ -0,0 +1,55 @@
+name: Test
+
+on:
+  push:
+    branches: [ core ]
+  pull_request:
+    branches: [ core ]
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        ruby-version:
+          - 2.3.8
+          - 2.4.5
+          - 2.5.3
+          - 2.6.3
+        gemfile:
+          - rails_4_2
+          - rails_5_0
+          - rails_5_1
+          - rails_5_2
+          - rails_6_0
+          - rails_6_1
+        exclude:
+          - ruby-version: '2.3.8'
+            gemfile: rails_6_0
+          - ruby-version: '2.3.8'
+            gemfile: rails_6_1
+          - ruby-version: '2.4.5'
+            gemfile: rails_6_0
+          - ruby-version: '2.4.5'
+            gemfile: rails_6_1
```

Comment:
I think that was neat how they made it work with ENV vars for travis. it should be possible to keep the same pattern with github actions, at least that's what I am trying